src/tools/robot-simu-python-module-py.cc

```diff
@@ -0,0 +1,17 @@
+#include <boost/python.hpp>
+#include "dynamic-graph/python/module.hh"
+#include <dynamic-graph/python/signal.hh>
+#include <sot/core/robot-simu.hh>
+#include <sot/core/flags.hh>
```

Comment:
I think we only need `dynamic-graph/python/module.hh` & `sot/core/robot-simu.hh`